scripts/wakeword/copy_lat_dir.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+# Copyright 2018-2020  Daniel Povey
+#           2018-2020  Yiming Wang
+
+utt_prefixes=
+max_jobs_run=30
+nj=75
+cmd=queue.pl
```

Comment:
`cmd` should be passed from local/chain/run_tdnn.sh.

Reply:
yes, and the default should always be run.pl, in these types of scripts.